src/status_im/subs/chats.cljs

```diff
@@ -358,3 +358,14 @@
    (re-frame/subscribe [:chats/chat chat-id]))
  :->
  :image)
+
+(re-frame/reg-sub
+ :group-chat/manage-members-count
+ (fn [[_ chat-id]] [(re-frame/subscribe [:chats/chat-by-id chat-id])
+                    (re-frame/subscribe [:group-chat/selected-participants])
+                    (re-frame/subscribe [:group-chat/deselected-members])])
```

Comment:
This code style for functions is not common, please fix it

Reply:
I have updated the subscription to use `:<-` syntax. Please let me know if you had something else in mind.

Reply:
Hey @ajayesivan !

I meant about code formatting. The code style for functions is:
```clojure
(fn [p1 p2 ...]
  'function-body-1
  'function-body-2)

;; and NOT:
(fn [p1 p2 ...] 'function-body-1
                'function-body-2)
```

Reply:
Sorry for not being clear enough with the first comment

Reply:
Thank you for the clarification!